index.html

```diff
@@ -11,17 +11,15 @@
 
 <body>
     <div class="background">
-        <div class="toolbar">
-            <div class="welcome">Welcome</div>
-            <nav>
-                <img class="menu" src="./assets/menu.svg" alt="menu">
-            </nav>
-        </div>
-        <h1 class="intro primary-color">Hey there. I’m Rex.</h1>
-        <h2 class="intro secondary-color">I am a full stack developer.</h2>
-        <p> I can help you build a product , feature or website.</p>
-        <p> Look through some of my work and experience!</p>
-        <p> If you like what you see and have a project you need coded, don’t hestiate to contact me. 😉</p>
+        <div class="flex">
+            <div class="welcome font">Welcome</div>
+            <img class="menu" src="./assets/menu.svg" alt="menu">
+        </div>
```

Comment:
Kindly use a `<header>` tag to wrap your header instead of a `<div>`. This is for semantic reasons and to improve your site SEO score.